src/Nethermind/Nethermind.Runner/Ethereum/Steps/InitializeNodeStats.cs

```diff
@@ -0,0 +1,42 @@
+﻿//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+using System.Threading.Tasks;
+using Nethermind.Runner.Ethereum.Context;
+using Nethermind.Stats;
+
+namespace Nethermind.Runner.Ethereum.Steps
+{
+    [RunnerStepDependency]
```

Comment:
this attribute has no meanings without any dependency type

Reply:
Technically yes. But my intention here is to show that this isn't an omission but we deliberately set no dependencies. So next person who touches it won't have to guess.